docs/cmd/tkn_bundle_list.md

```diff
@@ -23,7 +23,7 @@ eg task).
 
 Authentication:
 	There are three ways to authenticate against your registry.
-	1. By default, your docker.config in your home directory is used.
+	1. By default, your docker.config in your home directory and podman's auth.json are used.
```

Comment:
It's a "and" or a "or" ? ie both are used or either of them?

Reply:
It's an "and". Both are used.

Reply:
I guess it should be first check in `.docker/config.json` for the creds, if it exists then use them if not then check for podman's auth.json.

Reply:
> I guess it should be first check in `.docker/config.json` for the creds, if it exists then use them if not then check for podman's auth.json.

Yes. That's the logic. Resolve creds in order and return once it's found. Do I need to add detailed explanation?